capstone/src/main/java/com/step902020/capstone/OrganizationController.java

```diff
@@ -142,4 +146,27 @@ public RedirectView deleteOrganizationEvent(@RequestParam("event-id") String eve
   public @ResponseBody byte[] getImage(@RequestParam("email") String email) throws IOException {
     return gcsstore.serveImage("step90-2020", "step90-2020.appspot.com", email);
   }
+
+  /**
+   * Add new review to event
+   * @param user current user
+   * @param text Review's text
+   * @param orgId Organization's datastore id
+   * @return Updated review list
+   */
+  @PostMapping("/new-org-review")
+  public List<Review> addReview(
+          CurrentUser user,
+          @RequestParam("text") String text,
+          @RequestParam("reviewedObjectId") Long orgId) throws IOException {
+
+    Organization organization = this.organizationRepository.findById(orgId).get();
+    Individual individual = this.individualRepository.findFirstByEmail(user.getEmail());
+    String individualName = individual.firstName + " " + individual.lastName;
+    String individualEmail = individual.email;
+    Review review = new Review(individualName, individualEmail, text);
+    organization.addReview(review);
+    this.organizationRepository.save(organization);
+    return organization.reviews;
+  }
```

Comment:
Not sure if there is a bug or if it is because of data inconsistency issues, but when I enter a review for an org it does not actually save.

Reply:
oh yeah since the organizations don't have a review array, a nullException error happen. I just added empty arrays to all the orgs!